semantic-codeql/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Revision history for semantic-codeql
+
+## 0.1.0.0 -- YYYY-mm-dd
+
+* First version. Released on an unsuspecting world.
```

Comment:
Do we want all these default files in here?

Reply:
Yeah, you can nix this (this is from the scaffolding tool).

Reply:
Removed in https://github.com/github/semantic/pull/507/commits/e5a1392924e62e056827ee1ba47abd0ec8de5399